.vib/airflow/cypress/cypress/integration/airflow_test_suite.js

```diff
@@ -0,0 +1,68 @@
+/// <reference types="cypress" />
+import { random } from "./utils";
+
+it("allows user to log in and log out", () => {
+  cy.login();
+  cy.contains("DAGs").should('be.visible');
+  cy.contains("div", "Invalid login").should("not.exist");
+
+  cy.visit("/logout").then(() => {
+    cy.clearCookies();
```

Comment:
As discussed with Jota, does this actually test the logout functionality?

Reply:
As discussed, I don't think it really does. Instead, I will replace this test with the "Import Variables" feature, which is far more interesting 🙂